libnmstate/nmstate.py

```diff
@@ -256,3 +287,19 @@ def _get_routes_from_plugins(plugins):
             ret[Route.RUNNING].extend(plugin_routes.get(Route.RUNNING, []))
             ret[Route.CONFIG].extend(plugin_routes.get(Route.CONFIG, []))
     return ret
+
+
+def _get_iface_types_by_name(iface_infos, name):
+    """
+    Return the type of all ifaces with valid type and specified name
+    """
+    iface_types = []
+    for iface_info in iface_infos:
+        iface_type = iface_info.get(Interface.TYPE, InterfaceType.UNKNOWN)
+        if (
+            iface_type != InterfaceType.UNKNOWN
```

Comment:
maybe keep the default None, it will be simpler.

Reply:
Was this missed?

Reply:
I leave it as it was as I have to handle two cases:
 * Type not defined.
 * Type been defined as `UNKNOWN` explicitly.

Using default None cannot handle the second case simply.